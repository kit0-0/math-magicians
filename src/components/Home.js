import React from 'react';

function Home() {
  return (
    <div className=" h-screen flex flex-col ml-4">
      <h1 className="text-4xl font-bold mt-8">Welcome to Math Magicians</h1>
      <p className="text-xl mt-4 w-screen">
        Math Magicians is a web application that provides a calculator for performing arithmetic
        calculations and a random quote generator to inspire and motivate you.
      </p>
      <p className="text-lg mt-4">
        Get started by exploring the calculator functionality or discovering insightful quotes.
      </p>
    </div>
  );
}

export default Home;
