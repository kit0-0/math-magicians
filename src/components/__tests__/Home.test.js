import React from 'react';
import { render } from '@testing-library/react';
import Home from '../Home';

describe('Home component', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });

  // Write more tests for the Home component
  // covering different scenarios and interactions.
});
