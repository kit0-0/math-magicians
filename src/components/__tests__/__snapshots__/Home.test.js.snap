// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match the snapshot 1`] = `
<div>
  <div
    class=" h-screen flex flex-col ml-4"
  >
    <h1
      class="text-4xl font-bold mt-8"
    >
      Welcome to Math Magicians
    </h1>
    <p
      class="text-xl mt-4 w-screen"
    >
      Math Magicians is a web application that provides a calculator for performing arithmetic calculations and a random quote generator to inspire and motivate you.
    </p>
    <p
      class="text-lg mt-4"
    >
      Get started by exploring the calculator functionality or discovering insightful quotes.
    </p>
  </div>
</div>
`;
